plugins/extractit/icon_extractit/actions/extract_all/action.py

```diff
@@ -4,21 +4,35 @@
 # Custom imports below
 
 from icon_extractit.util.util import Regex
-from icon_extractit.util.extractor import extract, parse_time, clear_emails, extract_filepath, clear_urls, clear_domains
+from icon_extractit.util.extractor import (
+    extract,
+    parse_time,
+    clear_emails,
+    extract_filepath,
+    clear_urls,
+    clear_domains,
+    define_date_time_regex,
+)
 
 
 class ExtractAll(insightconnect_plugin_runtime.Action):
     def __init__(self):
         super(self.__class__, self).__init__(
-            name="extract_all", description=Component.DESCRIPTION, input=ExtractAllInput(), output=ExtractAllOutput()
+            name="extract_all",
+            description=Component.DESCRIPTION,
+            input=ExtractAllInput(),
+            output=ExtractAllOutput(),
         )
 
-    def run(self, params={}):
+    def run(self, params=None):
+        if params is None:
+            params = {}
```

Comment:
This should be reverted - empty dict as a default argument is correct

Reply:
Seconded ^ PyCharm will throw up a suggestion to change it to this but its best to ignore it

Reply:
Reverted changes to previous